src/main/java/za/co/neslotech/spring/trainspringbootinsight/model/User.java

```diff
@@ -0,0 +1,54 @@
+package za.co.neslotech.spring.trainspringbootinsight.model;
+
+import jakarta.persistence.*;
```

Comment:
We always try to avoid using wildcard imports otherwise you start importing every possible class from a library rather than what you need. You can change these settings in your IDE